src/Symfony/Component/PasswordHasher/Hasher/PasswordHasherFactoryInterface.php

```diff
@@ -26,7 +25,7 @@ interface PasswordHasherFactoryInterface
     /**
      * Returns the password hasher to use for the given user.
      *
-     * @param PasswordAuthenticatedUserInterface|UserInterface|string $user A PasswordAuthenticatedUserInterface/UserInterface instance or a class name
+     * @param PasswordHasherAwareInterface|PasswordAuthenticatedUserInterface|string $user A PasswordAuthenticatedUserInterface/PasswordHasherAwareInterface instance or a class name
```

Comment:
the description can be removed IMHO after the variable name

Reply:
I agree